.github/workflows/package-client.yml

```diff
@@ -280,15 +280,22 @@ jobs:
           sudo apt-get install -y fuse3 libfuse3-dev
         timeout-minutes: 2
 
-      - name: Install dependencies
-        shell: bash
-        run: |
-          npm clean-install
-          (cd electron && npm clean-install)
-          (cd ../bindings/electron && npm clean-install)
+      - name: Install client dependencies
+        # Use `--ignore-scripts` to prevent the postinstall script trying to be smarter than us and install electron dependencies.
```

Comment:
```suggestion
        # Use `--ignore-scripts` to prevent the postinstall script trying to be smarter than us and
        # install electron dependencies (as it would use `npm install` instead of `npm clean-install`).
```